pkgs/development/interpreters/ngn-k/default.nix

```diff
@@ -31,21 +31,27 @@ useStdenv.mkDerivation {
     # make self-reference for LICENSE
     substituteAllInPlace repl.k
```

Comment:
You can drop this line now, since we don't rely on `@out@` anymore to find the license in the new patch, right?

Reply:
Done.